Source/Core/InputCommon/ControllerEmu/Control/Input.h

```diff
@@ -12,7 +12,7 @@ namespace ControllerEmu
 class Input : public Control
 {
 public:
-  Input(const std::string& name, const std::string& ui_name);
-  explicit Input(const std::string& name);
+  Input(bool translate, const std::string& name, const std::string& ui_name);
```

Comment:
What do you think of making the bool an enum class instead to make things more descriptive (Translate/DoNotTranslate for example)?

Reply:
I've made it use an enum (without class).